### ISSUE TITLE
Consider formally separating API from frontend

### ISSUE THREAD

**Issue by createthis** [benmarten/CryptoETF#20]
I've been looking today for a general purpose JS library that would allow me to:

1. Grab coin market price in USD (or any other currency) from various exchanges via API
2. Grab coin balance by address (see #19 )

CryptoETF looks like it has functionality to achieve the first goal already. However, it has been packaged with a CLI frontend. It seems like it may reach a wider audience if the Node library API was moved into a separate project and documented.

This would allow reuse in more than one frontend. React web frontends, for example.

**Comment by barryvdh** [comment]
Yeah I was trying to fit it in a Vue project myself, but wasn't really working well. First step would be to seperate the Settings, as discussed here: https://github.com/benmarten/CryptoETF/pull/16#issuecomment-354599961
And then make the portfolio more extendible and publish this to npm.
